### DISCUSSION TITLE
Wyze Cam V2

### DISCUSSION THREAD

**Discussion by digitydogs** [gtxaspec/wz_mini_hacks#582] [Q&A]
So I cannot for the life of me seem to get this to work properly. 

Using the firmware version recommended in the wiki, manually downloaded. 

I can access the web configuration interface, but I cannot access any RTSP feeds, nor can i get anything related to go2rtc to show up anywhere. 

I have tried the /dev/video6 and video 7 endpoints (and all from 1-7), I have tried the various unicast based endpoints. 

I have tried with and without authentication. With authentication I just get asked for login info again and then connection refused. 

I have tried every combination of settings I can find in the issues and discussions threads and I am stuck. 

The screen shots others are sharing also have tabs and options that are nowhere to be found on my camera with the hacks.

Anyone have any ideas for me?  Here is the current config, I disabled go2rtc trying to just get the RTSP feeds working. 

 WEB_CAM_ENABLE=false
+ WEB_CAM_BIT_RATE=8000
+ WEB_CAM_FPS_RATE=25
+ CUSTOM_HOSTNAME=WCV3
+ ENABLE_USB_ETH=false
+ ENABLE_USB_ETH_MODULE_AUTODETECT=true
+ ENABLE_USB_ETH_MODULE_MANUAL=
+ USB_ETH_MAC_ADDR=
+ ENABLE_USB_DIRECT=false
+ USB_DIRECT_MAC_ADDR=02:01:02:03:04:08
+ ENABLE_USB_RNDIS=false
+ ENABLE_IPV6=false
+ ENABLE_WIREGUARD=false
+ WIREGUARD_IPV4=
+ WIREGUARD_PEER_ENDPOINT=
+ WIREGUARD_PEER_PUBLIC_KEY=
+ WIREGUARD_PEER_ALLOWED_IPS=
+ WIREGUARD_PEER_KEEP_ALIVE=
+ ENABLE_IPTABLES=false
+ ENABLE_NFSv4=false
+ ENABLE_RTL8189FS_DRIVER=true
+ ENABLE_ATBM603X_DRIVER=true
+ NTP_SERVER=
+ BONDING_ENABLED=false
+ BONDING_PRIMARY_INTERFACE=eth0
+ BONDING_SECONDARY_INTERFACE=wlan0
+ BONDING_LINK_MONITORING_FREQ_MS=100
+ BONDING_DOWN_DELAY_MS=5000
+ BONDING_UP_DELAY_MS=5000
+ BONDING_FAIL_OVER_MAC=0
+ REMOTE_SPOTLIGHT=false
+ RTSP_LOGIN=cams
+ RTSP_PASSWORD=cams
+ RTSP_PORT=8554
+ RTSP_HI_RES_ENABLED=true
+ RTSP_HI_RES_ENABLE_AUDIO=true
+ RTSP_LOW_RES_ENABLED=true
+ RTSP_LOW_RES_ENABLE_AUDIO=true
+ RTSP_AUTH_DISABLE=true
+ VIDEO_0_MAX_BITRATE=2048
+ VIDEO_0_TARGET_BITRATE=1024
+ VIDEO_0_ENC_PARAMETER=2
+ VIDEO_0_FPS=15
+ VIDEO_1_MAX_BITRATE=2048
+ VIDEO_1_TARGET_BITRATE=1024
+ VIDEO_1_ENC_PARAMETER=2
+ VIDEO_1_FPS=30
+ GO2RTC_SERVER_ENABLED=false
+ GO2RTC_HI_RES_ENABLED=false
+ GO2RTC_HI_RES_ENABLE_AUDIO=false
+ GO2RTC_LOW_RES_ENABLED=false
+ GO2RTC_LOW_RES_ENABLE_AUDIO=false
+ RTMP_STREAM_ENABLED=false
+ RTMP_STREAM_FEED=video1_unicast
+ RTMP_STREAM_SERVICE=youtube
+ RTMP_STREAM_DISABLE_AUDIO=false
+ RTMP_STREAM_YOUTUBE_KEY=
+ RTMP_STREAM_TWITCH_KEY=
+ RTMP_STREAM_FACEBOOK_KEY=
+ LIBCALLBACK_ENABLE=true
+ ENABLE_SWAP=true
+ ENABLE_USB_STORAGE=false
+ ENABLE_EXT4=false
+ ENABLE_CIFS=false
+ DISABLE_FW_UPGRADE=false
+ AUDIO_PROMPT_VOLUME=70
+ ENABLE_MP4_WRITE=false
+ NIGHT_DROP_DISABLE=false
+ DISABLE_MOTOR=false
+ ENABLE_FSCK_ON_BOOT=false
+ ENABLE_CAR_DRIVER=false
+ ENABLE_LOCAL_DNS=false
+ ENABLE_CRONTAB=false
+ ENABLE_SELFHOSTED_MODE=false
+ DEBUG_ENABLED=false
+ DEBUG_INITRAMFS_ENABLED=false
+ DEBUG_PASSWORD=false
+ ENABLE_SYSLOG_SAVE=false
+ WEB_SERVER_ENABLED=true
+ WEB_SERVER_AUTH=false
+ WEB_SERVER_LOGIN=cams
+ WEB_SERVER_PASSWORD=cams
+ CUSTOM_SCRIPT_PATH=

**Comment by digitydogs** [comment]
Diagnostic report:

Diagnostics T20(WYZEC1-JZ) on 9f242faa1f running wz_mini Wed Jul 19 09:00:46 PM PDT 2023 as WCV3
SD Card Test
no SD card error
no SD read only error
no file system error
no SD card (tf_prepare) error
no SD card health fail error
Firmware Version Test
Firmware Version: 9f242faa1f
[    0.000000] Initializing cgroup subsys cpu
[    0.000000] Initializing cgroup subsys cpuacct
[    0.000000] Linux version 3.10.14 (t20@mips-development) (gcc version 4.7.2 (Ingenic r2.3.3 2016.12) ) #47 PREEMPT Wed Jul 13 00:36:41 PDT 2022
[    0.000000] bootconsole [early0] enabled
[    0.000000] CPU0 RESET ERROR PC:8006A22C
[    0.000000] [<8006a22c>] 0x8006a22c
[    0.000000] CPU0 revision is: 00d00101 (Ingenic Xburst)
[    0.000000] FPU revision is: 00b70000
[    0.000000] cgu_get_rate, parent = 860160000, rate = 0, m = 0, n = 0, reg val = 0x000020ff
[    0.000000] CCLK:860MHz L2CLK:430Mhz H0CLK:200MHz H2CLK:200Mhz PCLK:100Mhz
[    0.000000] Determined physical RAM map:
[    0.000000]  memory: 004d0000 @ 00010000 (usable)
[    0.000000]  memory: 00070000 @ 004e0000 (usable after init)
[    0.000000] User-defined physical RAM map:
[    0.000000]  memory: 06800000 @ 00000000 (usable)
[    0.000000] Initrd not found or empty - disabling initrd
[    0.000000] Zone ranges:
[    0.000000]   Normal   [mem 0x00000000-0x067fffff]
[    0.000000] Movable zone start for each node
[    0.000000] Early memory node ranges
[    0.000000]   node   0: [mem 0x00000000-0x067fffff]
[    0.000000] On node 0 totalpages: 26624
[    0.000000] free_area_init_node: node 0, pgdat 804dd3d0, node_mem_map 81000000
[    0.000000]   Normal zone: 208 pages used for memmap
[    0.000000]   Normal zone: 0 pages reserved
[    0.000000]   Normal zone: 26624 pages, LIFO batch:7
[    0.000000] Primary instruction cache 32kB, 8-way, VIPT, linesize 32 bytes.
[    0.000000] Primary data cache 32kB, 8-way, VIPT, no aliases, linesize 32 bytes
[    0.000000] pls check processor_id[0x00d00101],sc_jz not support!
[    0.000000] MIPS secondary cache 128kB, 8-way, linesize 32 bytes.
[    0.000000] pcpu-alloc: s0 r0 d32768 u32768 alloc=1*32768
[    0.000000] pcpu-alloc: [0] 0 
[    0.000000] Built 1 zonelists in Zone order, mobility grouping off.  Total pages: 26416
[    0.000000] Kernel command line: console=ttyS1,115200n8 mem=104M@0x0 ispmem=8M@0x6800000 rmem=16M@0x7000000 init=/init mtdparts=jz_sfc:256k(boot),2048k(kernel),3392k(root),640k(driver),4736k(appfs),2048k(backupk),640k(backupd),2048k(backupa),256k(config),256k(para),-(flag)
[    0.000000] PID hash table entries: 512 (order: -1, 2048 bytes)
[    0.000000] Dentry cache hash table entries: 16384 (order: 4, 65536 bytes)
[    0.000000] Inode-cache hash table entries: 8192 (order: 3, 32768 bytes)
[    0.000000] Memory: 99484k/106496k available (4166k kernel code, 7012k reserved, 760k data, 448k init, 0k highmem)
[    0.000000] SLUB: HWalign=32, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
[    0.000000] Preemptible hierarchical RCU implementation.
[    0.000000] NR_IRQS:418
[    0.000000] clockevents_config_and_register success.
[    0.000024] Calibrating delay loop... 858.52 BogoMIPS (lpj=4292608)
[    0.087753] pid_max: default: 32768 minimum: 301
[    0.092742] Mount-cache hash table entries: 512
[    0.097850] Initializing cgroup subsys debug
[    0.102112] Initializing cgroup subsys freezer
[    0.107931] devtmpfs: initialized
[    0.112615] regulator-dummy: no parameters
[    0.116978] NET: Registered protocol family 16
[    0.126814] bio: create slab  at 0
[    0.133142] jz-dma jz-dma: JZ SoC DMA initialized
[    0.138273] usbcore: registered new interface driver usbfs
[    0.143805] usbcore: registered new interface driver hub
[    0.149320] usbcore: registered new device driver usb
[    0.154592] i2c-gpio i2c-gpio.1: using pins 57 (SDA) and 58 (SCL)
[    0.160854]  (null): set:249  hold:250 dev=100000000 h=500 l=500
[    0.166986] media: Linux media interface: v0.10
[    0.171538] Linux video capture interface: v2.00
[    0.176426] Advanced Linux Sound Architecture Driver Initialized.
[    0.184414] Switching to clocksource jz_clocksource
[    0.189366] cfg80211: Calling CRDA to update world regulatory domain
[    0.196748] dwc2 otg probe start
[    0.196784] jz-dwc2 jz-dwc2: cgu clk gate get error
[    0.201670] DWC IN OTG MODE
[    0.205192] dwc2 dwc2: Keep PHY ON
[    0.208585] dwc2 dwc2: Using Buffer DMA mode
[    0.212893] dwc2 dwc2: Core Release: 3.00a
[    0.217093] dwc2 dwc2: DesignWare USB2.0 High-Speed Host Controller
[    0.223435] dwc2 dwc2: new USB bus registered, assigned bus number 1
[    0.230940] hub 1-0:1.0: USB hub found
[    0.234690] hub 1-0:1.0: 1 port detected
[    0.238838] dwc2 dwc2: DWC2 Host Initialized
[    0.243367] dwc2 dwc2: enter dwc2_gadget_plug_change:2589: plugin = 1 pullup_on = 0 suspend = 0
[    0.252205] dwc2 otg probe success
[    0.252397] NET: Registered protocol family 2
[    0.257393] TCP established hash table entries: 1024 (order: 1, 8192 bytes)
[    0.264418] TCP bind hash table entries: 1024 (order: 0, 4096 bytes)
[    0.270904] TCP: Hash tables configured (established 1024 bind 1024)
[    0.277350] TCP: reno registered
[    0.280550] UDP hash table entries: 256 (order: 0, 4096 bytes)
[    0.286502] UDP-Lite hash table entries: 256 (order: 0, 4096 bytes)
[    0.293104] NET: Registered protocol family 1
[    0.297824] RPC: Registered named UNIX socket transport module.
[    0.303757] RPC: Registered udp transport module.
[    0.308591] RPC: Registered tcp transport module.
[    0.313292] RPC: Registered tcp NFSv4.1 backchannel transport module.
[    0.319852] dwc2 dwc2: ID PIN CHANGED!
[    0.562422] freq_udelay_jiffys[0].max_num = 10
[    0.566948] cpufreq 	udelay 	loops_per_jiffy	
[    0.571286] 12000	 59885	 59885	
[    0.574552] 24000	 119771	 119771	
[    0.578028] 60000	 299428	 299428	
[    0.581439] 120000	 598857	 598857	
[    0.584970] 200000	 998095	 998095	
[    0.588626] 300000	 1497142	 1497142	
[    0.592257] 600000	 2994285	 2994285	
[    0.595980] 792000	 3952457	 3952457	
[    0.599673] 1008000	 5030400	 5030400	
[    0.603470] 1200000	 5988571	 5988571	
[    0.612787] squashfs: version 4.0 (2009/01/31) Phillip Lougher
[    0.619718] jffs2: version 2.2. (NAND) Â© 2001-2006 Red Hat, Inc.
[    0.626310] msgmni has been set to 194
[    0.631483] io scheduler noop registered
[    0.635420] io scheduler cfq registered (default)
[    0.643078] jz-uart.1: ttyS1 at MMIO 0x10031000 (irq = 58) is a uart1
[    0.651362] console [ttyS1] enabled, bootconsole disabled
[    0.667571] brd: module loaded
[    0.673301] loop: module loaded
[    0.677450] zram: Created 2 device(s) ...
[    0.681745] logger: created 256K log 'log_main'
[    0.687172] jz SADC driver registeres over!
[    0.692933] jz TCU driver register completed
[    0.697868] the id code = c84018, the flash name is GD25Q127C
[    0.703829] JZ SFC Controller for SFC channel 0 driver register
[    0.710047] 11 cmdlinepart partitions found on MTD device jz_sfc
[    0.716290] Creating 11 MTD partitions on "jz_sfc":
[    0.721350] 0x000000000000-0x000000040000 : "boot"
[    0.727041] 0x000000040000-0x000000240000 : "kernel"
[    0.732845] 0x000000240000-0x000000590000 : "root"
[    0.738539] 0x000000590000-0x000000630000 : "driver"
[    0.744363] 0x000000630000-0x000000ad0000 : "appfs"
[    0.750163] 0x000000ad0000-0x000000cd0000 : "backupk"
[    0.756133] 0x000000cd0000-0x000000d70000 : "backupd"
[    0.762061] 0x000000d70000-0x000000f70000 : "backupa"
[    0.768043] 0x000000f70000-0x000000fb0000 : "config"
[    0.773923] 0x000000fb0000-0x000000ff0000 : "para"
[    0.779643] 0x000000ff0000-0x000001000000 : "flag"
[    0.785265] SPI NOR MTD LOAD OK
[    0.789101] jzmmc_v1.2 jzmmc_v1.2.0: vmmc regulator missing
[    0.825944] jzmmc_v1.2 jzmmc_v1.2.0: register success!
[    0.831346] jzmmc_v1.2 jzmmc_v1.2.1: vmmc regulator missing
[    0.876868] jzmmc_v1.2 jzmmc_v1.2.1: register success!
[    0.882353] hidraw: raw HID events driver (C) Jiri Kosina
[    0.888303] usbcore: registered new interface driver usbhid
[    0.894078] usbhid: USB HID core driver
[    0.903611] Netfilter messages via NETLINK v0.30.
[    0.908811] TCP: cubic registered
[    0.912799] NET: Registered protocol family 10
[    0.918442] NET: Registered protocol family 17
[    0.924185] input: gpio-keys as /devices/platform/gpio-keys/input/input0
[    0.931546] drivers/rtc/hctosys.c: unable to open rtc device (rtc0)
[    0.938173] ALSA device list:
[    0.941251]   #0: Loopback 1
[    0.944233]   #2: Loopback 2
[    0.949917] Freeing unused kernel memory: 448K (804e0000 - 80550000)
[    2.155829] jzmmc_v1.2 jzmmc_v1.2.0: card inserted, state=0
[    3.399796] wait stable.[246][cgu_msc0]
[    3.403822] mmc0: new high speed SDHC card at address aaaa
[    3.409840] mmcblk0: mmc0:aaaa SU04G 3.69 GiB 
[    3.430221]  mmcblk0: p1
[    5.906878] v4l2loopback driver version 0.12.5 loaded
[    6.745997] Adding 131068k swap on /opt/wz_mini/swap.  Priority:-1 extents:1 across:131068k SS
[    8.643332] register all isp device successfully!
[    8.649879] @@@@ tx-isp-probe ok @@@@@
[    8.694892] exFAT: Version 1.2.9
[    8.731257] motor_probe [ChenX @ Hualai Technologies]
[    8.737586] Hualai motor driver version: MT20180918g
[    8.753202] jz_pwm_probe[198] d_name = tcu_chn0
[    8.764050] request pwm channel 0 successfully
[    8.770331] pwm-jz pwm-jz: jz_pwm_probe register ok !
[    8.804576] jz_codec_register: probe() successful!
[    8.810728] cgu_set_rate, parent = 860160000, rate = 2048000, n = 420, reg val = 0x000021a4
[    8.810746] cgu_enable,cgu_i2s reg val = 0x200021a4
[    9.156234] dma dma0chan24: Channel 24 have been requested.(phy id 7,type 0x06 desc a4574000)
[    9.165489] dma dma0chan25: Channel 25 have been requested.(phy id 6,type 0x06 desc a459a000)
[    9.362701] wlan power on
[    9.421632] mmc1: card claims to support voltages below the defined range. These will be ignored.
[    9.446036] mmc1: new SDIO card at address 0001
[   18.683148] name : i2c-gpio1 nr : 1
[   18.689402] name : i2c0 nr : 0
[   18.753385]  sensor_read: addr=0xa value = 0xf
[   18.758495]  sensor_read: addr=0xb value = 0x23
[   18.763201] info: success sensor find : jxf23
[   18.767770] misc sinfo_release
[   19.390241] set sensor gpio as PA-low-10bit
[   19.394590] probe ok ------->jxf23
[   19.546728] -----jxf23_detect: 631 ret = 0, v = 0x0f
[   19.547232] -----jxf23_detect: 639 ret = 0, v = 0x23
[   19.547245] jxf23 0-0040: jxf23 chip found @ 0x40 (i2c0)
[   19.552752] tx_isp: Registered sensor subdevice jxf23 0-0040
[   19.558830] do not support ctrl->cmd ====1
[   19.558850] do not support ctrl->cmd ====0
[   19.562507] do not support ctrl->cmd ====1
[   19.837646] do not support ctrl->cmd ====1
[   19.838574] jxf23 stream on
[   20.056361] ###### image_tuning_v4l2_open 4219 #######
[   20.073233] before register 0x1f value : 0x01
[   20.075417] after register 0x1f value : 0x81
[   20.075438] do not support ctrl->cmd ====1
[   20.360317] &&& chan1  scaler.max_width = 1920 max_height = 1080  min_width = 128 min_height = 128 &&&
[   20.923061] &&& chan2  scaler.max_width = 800 max_height = 800  min_width = 128 min_height = 128 &&&
[   20.938149] codec_codec_ctl: set sample rate...
[   20.938725] codec_codec_ctl: set device...
[   21.335841] codec_set_device: set device: MIC...
[   21.534407] codec_codec_ctl: set repaly channel...
[   21.534451] codec_codec_ctl: set sample rate...
[   21.534774] codec_codec_ctl: set device...
[   21.765912] codec_set_device: set device: speaker...
[   23.138432] IPv6: ADDRCONF(NETDEV_UP): wlan0: link is not ready
[   24.451661] IPv6: ADDRCONF(NETDEV_UP): wlan0: link is not ready
[   37.152673] IPv6: ADDRCONF(NETDEV_CHANGE): wlan0: link becomes ready
[   43.136999] wz_mini done
[GitHub: wz_mini_hack](https://github.com/gtxaspec/wz_mini_hacks)
T20(WYZEC1-JZ)
9f242faa1f
Wed Jul 19 09:00:46 PM PDT 2023
 WCV3

**Comment by digitydogs** [comment]
Frigate logs attempting to access 4 different versions of the stream

023-07-26 18:48:38.943220452  [2023-07-26 18:48:38] watchdog.front_door            ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-26 18:48:38.943221893  [2023-07-26 18:48:38] ffmpeg.front_door.detect       ERROR   : [rtsp @ 0x560ba675b200] method DESCRIBE failed: 404 Stream Not Found
2023-07-26 18:48:38.943223300  [2023-07-26 18:48:38] ffmpeg.front_door.detect       ERROR   : rtsp://192.168.2.10:8554/video1_unicast - detect - record - rtmp: Server returned 404 Not Found
2023-07-26 18:48:38.961467103  [2023-07-26 18:48:38] watchdog.front_door2           ERROR   : Ffmpeg process crashed unexpectedly for front_door2.
2023-07-26 18:48:38.961470267  [2023-07-26 18:48:38] watchdog.front_door2           ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-26 18:48:38.961471686  [2023-07-26 18:48:38] ffmpeg.front_door2.detect      ERROR   : [rtsp @ 0x55b250e5f200] method DESCRIBE failed: 404 Stream Not Found
2023-07-26 18:48:38.961473083  [2023-07-26 18:48:38] ffmpeg.front_door2.detect      ERROR   : rtsp://192.168.2.10:8554/video2_unicast - detect - record - rtmp: Server returned 404 Not Found
2023-07-26 18:48:38.973731481  [2023-07-26 18:48:38] watchdog.front_door3           ERROR   : Ffmpeg process crashed unexpectedly for front_door3.
2023-07-26 18:48:38.973734682  [2023-07-26 18:48:38] watchdog.front_door3           ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-26 18:48:38.973736098  [2023-07-26 18:48:38] ffmpeg.front_door3.detect      ERROR   : [rtsp @ 0x563ac920a200] method DESCRIBE failed: 404 Stream Not Found
2023-07-26 18:48:38.973737517  [2023-07-26 18:48:38] ffmpeg.front_door3.detect      ERROR   : rtsp://192.168.2.10:8554/dev/video6 - detect - record - rtmp: Server returned 404 Not Found
2023-07-26 18:48:38.993628265  [2023-07-26 18:48:38] watchdog.front_door4           ERROR   : Ffmpeg process crashed unexpectedly for front_door4.
2023-07-26 18:48:38.993631398  [2023-07-26 18:48:38] watchdog.front_door4           ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-26 18:48:38.993632857  [2023-07-26 18:48:38] ffmpeg.front_door4.detect      ERROR   : [rtsp @ 0x561ddcc64200] method DESCRIBE failed: 404 Stream Not Found
2023-07-26 18:48:38.993634296  [2023-07-26 18:48:38] ffmpeg.front_door4.detect      ERROR   : rtsp://192.168.2.10:8554/dev/video7 - detect - record - rtmp: Server returned 404 Not Found

Trying with only hi or low res enabled results in the same as above and also the bare unicast url
2023-07-26 18:51:46.525692216  [2023-07-26 18:51:46] watchdog.front_door            ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-26 18:51:46.525695849  [2023-07-26 18:51:46] ffmpeg.front_door.detect       ERROR   : [rtsp @ 0x55da32f45200] method DESCRIBE failed: 404 Stream Not Found
2023-07-26 18:51:46.525705549  [2023-07-26 18:51:46] ffmpeg.front_door.detect       ERROR   : rtsp://192.168.2.10:8554/unicast - detect - record - rtmp: Server returned 404 Not Found

**Comment by RyanHope** [comment]
I've never gotten this project to work either.

**Reply by digitydogs** [reply]
I now have. 

Reflash your card with a fresh setup and do NOT enable either of the RTSP feeds. Enable the WebUI and ALL of the go2rtc options. 

Goto the go2rtc ui mentioned below and then under the config ensure the /dev/video* is correct for your cam (they are listed in a comment in the config itsellf)  restart and click on links next to the stream for all your url  and type options 

Once you ensure you can access your streams you can use the webui to disable the wyze server access or make other changes if needed, and tweak any go2rtc settings if you so desire.

**Reply by supermarkert** [reply]
@digitydogs 
Do you mind sharing your go2rtc config from your V2? I'm not following the /dev/video6 instructions, and I'm feeling dumb.

Edit: Nevermind! I was dumb, and I eventually figured it out. Seems to be working great now!

**Comment by gtxaspec** [comment]
see https://github.com/gtxaspec/wz_mini_hacks/wiki/Configuration-File#go2rtc-streaming-server

when you enabled go2rtc, were you able to access the UI at :1984?

**Reply by digitydogs** [reply]
I could not so I went ahead and re-flashed the card and started over from scratch with just go2rtc and webui enabled.

After that I was able to access the go2rtc interface and adjust the config for my cam. 

I now have fully working streams!  Thank you for all your work and your quick help!

**Comment by digitydogs** [comment]
This is my current go2rtc config:

  720p:
# NOTE: USE /dev/video6 for Wyze Cam v2/PanV1!
  - exec:ffmpeg -fflags +genpts+flush_packets+nobuffer+bitexact -flags
+low_delay -thread_queue_size 100 -analyzeduration 50 -probesize 50
-pix_fmt yuv420p -r 30 -i /dev/video6  -flags +global_header -movflags
+empty_moov+default_base_moof+frag_keyframe -framerate 10 -video_size
1280x720 -pix_fmt yuv420p -c:v copy -copyts -rtsp_transport tcp -f rtsp
{output}
  - exec:ffmpeg -hide_banner -f alsa -ac 1 -i dsnooper0 -c:a copy -muxdelay
0 -muxpreload 0 -f rtsp {output}
  - exec:ffmpeg -hide_banner -f alsa -ac 1 -i dsnooper0 -c:a libfdk_aac
-afterburner 1 -channels 1 -b:a 128k -profile:a aac_he -ar 16000 -strict
experimental -muxdelay 0 -muxpreload 0 -ac:a 1 -f rtsp {output}
# libopus has higher cpu requirements, audio may studder.
#  - exec:ffmpeg -hide_banner -f alsa -ac 1 -i dsnooper0 -c:a libopus
-muxdelay 0 -muxpreload 0 -ac:a 1 -af adelay=0|0 -f rtsp {output}
  480p:
# NOTE: USE /dev/video7 for Wyze Cam v2/PanV1!
  - exec:ffmpeg -fflags +genpts+flush_packets+nobuffer+bitexact -flags
+low_delay -thread_queue_size 100 -analyzeduration 50 -probesize 50
-pix_fmt yuv420p -r 30 -i /dev/video7 -framerate 10 -video_size 640x480
-pix_fmt yuv420p -c:v copy -copyts -rtsp_transport tcp -f rtsp {output}
  - exec:ffmpeg -hide_banner -f alsa -ac 1 -i dsnooper2 -c:a copy -muxdelay
0 -muxpreload 0 -ac:a 1 -f rtsp {output}
  - exec:ffmpeg -hide_banner -f alsa -ac 1 -i dsnooper2 -c:a libfdk_aac
-afterburner 1 -channels 1 -b:a 128k -profile:a aac_he -ar 16000 -strict
experimental -muxdelay 0 -muxpreload 0 -ac:a 1 -f rtsp {output}
# libopus has higher cpu requirements, audio may studder.
#  - exec:ffmpeg -hide_banner -f alsa -ac 1 -i dsnooper2 -c:a libopus
-muxdelay 0 -muxpreload 0 -ac:a 1 -af adelay=0|0 -f rtsp {output}

**Reply by supermarkert** [reply]
Thanks. 
So, first thoughts: is this still for the V2? Any reason you didn't stick with the default 1080/360?
FWIW, everything is working perfectly on my end, except I can't view clips from my Android phone. Gives me corrupted video errors. But I can view non-Wyze clips fine, or view Wyze clips from any other desktop browser. Something seems to be funky about the files that my phone can't handle, and it's infuriating.

**Reply by rgviewer** [reply]
I was wondering if you were able to install wyze_mini_hacks on your V2 recently. I followed the instructions and I can not get the camera to boot up after putting in sd card with demo.bin and sd_root folder. It just blinks blue and yellow. If I try to add the camera in wyze app it says "can not find specified network name".

**Reply by Tom8867** [reply]
Copy the complete content of **SD Card** folder on your SD Card, this means in the root of your SD Card is:
factory_t31_ZMC6tiIDQN and the Folder wz_mini

**Reply by rgviewer** [reply]
OMG thank you so much Tom8867. I have been pulling my hair out flashing and reflashing. I can't believe I missed something so simple as this. Was about to give up. Can't thank you enough. still work to do but I think I got it now.

**Comment by digitydogs** [comment]
My cams (8) are fed through NVR and a coral and used to trigger various
automations , and those are the resolutions I have found to work best for
file size while retaining high accuracy
